autopush/db.py

```diff
@@ -1078,6 +1117,21 @@ def create_initial_message_tables(self):
         an entry for tomorrow, if tomorrow is a new month.
 
         """
+        if not self.allow_table_rotation:
+            tablename = self.resource.get_latest_message_tablename(
```

Comment:
If I understand correctly, `self._message_conf.tablename` is the config option to specify the root name for the message table. Here, it's used as a mask to determine the message table from the returned list of tables.

the following table creation methods will probably not be called outside of test routines, since the table will have already been created.

Reply:
my fault, I misunderstood that we'd be going w/ the final dated table after toggling allow_table_rotation as False